test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_config_test.clj

```diff
@@ -4,11 +4,11 @@
   (:require [clojure.test :refer :all]
             [clojure.java.io :refer [resource]]
             [me.raynes.fs :as fs]
-            [schema.core :as schema]
```

Comment:
This made me realize that we've started to turn on schema validation in our tests.  If you wanted to it'd just be:
```clj
(:require [schema.test :as schema-test])
(use-fixtures :once schema-test/validate-schema)
```